Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -155,16 +155,18 @@ open class ReadingThemesControlsViewController: UIViewController {
             fallthrough
         case Theme.dark.name:
             imageDimmingSwitch.isEnabled = true
-            makeImageDimmingSwitchMoreProminentForDisabledState(imageDimmingSwitch.isEnabled)
+            isImageDimmingSwitchEnabled = imageDimmingSwitch.isEnabled
             applyBorder(to: darkThemeButton)
         default:
             break
         }
     }
     
-    func makeImageDimmingSwitchMoreProminentForDisabledState(_ isEnabled: Bool) {
-        imageDimmingSwitch.layer.cornerRadius = imageDimmingSwitch.frame.height / 2
-        imageDimmingSwitch.backgroundColor = isEnabled ? nil : UIColor.wmf_lighterGray
+    var isImageDimmingSwitchEnabled = true {
+        didSet {
+            imageDimmingSwitch.layer.cornerRadius = imageDimmingSwitch.frame.height / 2
```

Comment:
you can now add `imageDimmingSwitch.isEnabled = isImageDimmingSwitchEnabled` here and remove both lines above